pom.xml

```diff
@@ -69,7 +69,7 @@
         <pluginRepository>
             <snapshots>
                 <enabled>false</enabled>
-            </snapshots>
+           </snapshots>
```

Comment:
Oof, it looks like you forgot to revert the change to this line.  You accidentally removed one of the spaces..

Reply:
Hopefully fixed.